app/hostpathplugin/Dockerfile

```diff
@@ -1,6 +1,9 @@
+#
+# Use ./hack/build-container-hostpathplugin.sh from top of the tree
+#
 FROM alpine
```

Comment:
Is it really necessary or desirable to use alpine as base?

The current hostpathplugin binary is completely static, so shouldn't it also run when the image is entirely empty otherwise?